universal-application-tool-0.0.1/app/views/components/TextFormatter.java

```diff
@@ -0,0 +1,120 @@
+package views.components;
+
+import static j2html.TagCreator.a;
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.li;
+import static j2html.TagCreator.text;
+import static j2html.TagCreator.ul;
+
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
+import com.linkedin.urls.Url;
+import com.linkedin.urls.detection.UrlDetector;
+import com.linkedin.urls.detection.UrlDetectorOptions;
+import j2html.tags.ContainerTag;
+import j2html.tags.DomContent;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import views.style.Styles;
+
+/**
+ * The TextFormatter class introduces options for converting plain-text strings into richer HTML
+ * content.
+ *
+ * <p>The following is currently supported:
+ *
+ * <ul>
+ *   <li>Accordions
```

Comment:
Can you add to the description so its clearer what is supported (e.g. lines with `###` and `*`).

Reply:
Done.